plugins/inventory/foreman.py

```diff
@@ -85,6 +85,60 @@
         description: Number of hosts per batch that will be retrieved from the Foreman API per individual call
         type: integer
         default: 250
+      foreman:
+        use_reports_api:
+            description: Use Reporting API
+            type: boolean
+            default: False
+      report:
```

Comment:
```suggestion
      report:
        description: report API specific configuration
        type: dict
```

Reply:
(that's what the test failure "ERROR! Unable to retrieve documentation from 'theforeman.foreman.foreman' due to: All (sub-)options and return values must have a 'description' field" is about)

Reply:
I wasn't sure what was happening with the test failures. Thanks..Updated.